.gitignore

```diff
@@ -4,3 +4,5 @@ dist/
 cover.out
 .idea/
 .release-env
+.kdev4/
+sdpctl.kdev4
```

Comment:
What are these for, and should they be commited?

Reply:
They are for my IDE, Yes they should be like all the other things to ignore stuff from other IDE.